visuals/Iconography.html

```diff
@@ -0,0 +1,210 @@
+	<span class="ws-icon-chip-text">Notes</span>
+	<span class="ws-icon-chip-text">e911</span>
+</div>
+
+<div class="ws-icon-chip">
+	<span class="ws-icon-chip-icon"><i class="icon-reload"></i></span>
+	<span class="ws-icon-chip-text">Reload</span>
+	<span class="ws-icon-chip-text">e911</span>
+</div>
+
+<div class="ws-icon-chip">
+	<span class="ws-icon-chip-icon"><i class="icon-remove"></i></span>
+	<span class="ws-icon-chip-text">Remove</span>
+	<span class="ws-icon-chip-text">e916</span>
+</div>
+
+<div class="ws-icon-chip">
+	<span class="ws-icon-chip-icon"><i class="icon-search"></i></span>
+	<span class="ws-icon-chip-text">Search</span>
+	<span class="ws-icon-chip-text">e917</span>
+</div>
+
+<div class="ws-icon-chip">
+	<span class="ws-icon-chip-icon"><i class="icon-servers"></i></span>
+	<span class="ws-icon-chip-text">Servers</span>
+	<span class="ws-icon-chip-text">e918</span>
+</div>
+
+<div class="ws-icon-chip">
+	<span class="ws-icon-chip-icon"><i class="icon-sort-down"></i></span>
+	<span class="ws-icon-chip-text">Sort Down</span>
+	<span class="ws-icon-chip-text">e919</span>
+</div>
+
+<div class="ws-icon-chip">
+	<span class="ws-icon-chip-icon"><i class="icon-sort-up"></i></span>
+	<span class="ws-icon-chip-text">Sort Up</span>
+	<span class="ws-icon-chip-text">e91a</span>
+</div>
+
+<div class="ws-icon-chip">
+	<span class="ws-icon-chip-icon"><i class="icon-sort"></i></span>
+	<span class="ws-icon-chip-text">Sort</span>
+	<span class="ws-icon-chip-text">e91b</span>
+</div>
+
+<div class="ws-icon-chip">
+	<span class="ws-icon-chip-icon"><i class="icon-support"></i></span>
+	<span class="ws-icon-chip-text">Support</span>
+	<span class="ws-icon-chip-text">e91c</span>
+</div>
+
+<div class="ws-icon-chip">
+	<span class="ws-icon-chip-icon"><i class="icon-tag"></i></span>
+	<span class="ws-icon-chip-text">Tag</span>
+	<span class="ws-icon-chip-text">e91d</span>
+</div>
+
+<div class="ws-icon-chip">
+	<span class="ws-icon-chip-icon"><i class="icon-ticket"></i></span>
+	<span class="ws-icon-chip-text">Ticket</span>
+	<span class="ws-icon-chip-text">e91e</span>
+</div>
+
+<div class="ws-icon-chip">
+	<span class="ws-icon-chip-icon"><i class="icon-user"></i></span>
+	<span class="ws-icon-chip-text">User</span>
+	<span class="ws-icon-chip-text">e91f</span>
+</div>
+
+<div class="ws-icon-chip">
+	<span class="ws-icon-chip-icon"><i class="icon-warning"></i></span>
+	<span class="ws-icon-chip-text">Warning</span>
+	<span class="ws-icon-chip-text">e920</span>
+</div>
+
+<style type="text/css">
+	.ws-icon-chip{
+		padding-bottom: 32px;
+	}
+
+	.ws-icon-chip-icon{
+		color:#757575;
+		font-size: 32px;
+		text-align: center;
+	}
+
+	.ws-icon-chip-icon, .ws-icon-chip-text{
+		display: block;
+	}
+
+  .ws-icon-chip-text {
+    border-top: 1px solid #e0e0e0;
+    padding:8px 0 0 0;
+  }
+
+  .ws-icon-chip-text:last-of-type {
+  	padding: 0 0 8px 0;
+  	border-top: 0;
+    border-bottom: 1px solid #e0e0e0;
+  }
+
+</style>
```

Comment:
One thing you could consider here is using yaml data to hold all the class names and character codes, then setting up a jekyll loop to iterate over all those, rather than writing html by hand for each one.